pkgs/tools/misc/ntfy-sh/default.nix

```diff
@@ -15,8 +21,19 @@ buildGoModule rec {
 
   doCheck = false;
 
+  nativeBuildInputs = [
+    debianutils
+    mkdocs
+    nodejs
+    python3
+    python3Packages.mkdocs-material
+    python3Packages.mkdocs-minify
+  ];
+
   preBuild = ''
-    make cli-deps-static-sites
+    sed -i 's# /bin/echo# echo#' Makefile
```

Comment:
Can you move this to `postPatch`? In my opinion, that's more semantically correct with relation to the phases. (If you'd rather not, let me know -- happy to discuss, as it really doesn't matter in the grand scheme of things.)

Reply:
Agreed, that seems more logical indeed.
And no sorry needed, I'm learning on the go. :)